plugins/woocommerce-admin/client/marketing/overview-multichannel/Channels/RegisteredChannelCardBody.tsx

```diff
@@ -0,0 +1,159 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import GridiconCheckmarkCircle from 'gridicons/dist/checkmark-circle';
+import GridiconSync from 'gridicons/dist/sync';
+import GridiconNotice from 'gridicons/dist/notice';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { PluginCardBody } from '~/marketing/components';
+import { RegisteredChannel, SyncStatusType } from '~/marketing/types';
+import './RegisteredChannelCardBody.scss';
+
+type RegisteredChannelCardBodyProps = {
+	registeredChannel: RegisteredChannel;
+};
+
+type SyncStatusPropsType = {
+	status: SyncStatusType;
+};
+
+const iconSize = 18;
+const className = 'woocommerce-marketing-sync-status';
+
+const SyncStatus: React.FC< SyncStatusPropsType > = ( { status } ) => {
+	if ( status === 'failed' ) {
+		return (
+			<div
+				className={ classnames( className, `${ className }__failed` ) }
+			>
+				<GridiconNotice size={ iconSize } />
+				{ __( 'Sync failed', 'woocommerce' ) }
+			</div>
+		);
+	}
+
+	if ( status === 'syncing' ) {
+		return (
+			<div
+				className={ classnames( className, `${ className }__syncing` ) }
+			>
+				<GridiconSync size={ iconSize } />
+				{ __( 'Syncing', 'woocommerce' ) }
+			</div>
+		);
+	}
+
+	return (
+		<div className={ classnames( className, `${ className }__synced` ) }>
+			<GridiconCheckmarkCircle size={ iconSize } />
+			{ __( 'Synced', 'woocommerce' ) }
+		</div>
+	);
+};
+
+type IssueStatusPropsType = {
+	registeredChannel: RegisteredChannel;
+};
+
+const issueStatusClassName = 'woocommerce-marketing-issue-status';
```

Comment:
Same as https://github.com/woocommerce/woocommerce/pull/36541/files#r1088667040

I would move this at the top